core/probePlugin/src/main/scala/org/virtuslab/ideprobe/Screenshot.scala

```diff
@@ -9,18 +9,15 @@ object Screenshot {
   private val id = new AtomicInteger(0)
 
   private val probeDisplay = sys.env.get("DISPLAY")
-  private val outputDirectory = for {
-    screenshotsDir <- sys.env.get("IDEPROBE_SCREENSHOTS_DIR")
-    testSuite <- sys.env.get("IDEPROBE_TEST_SUITE")
-    testCase <- sys.env.get("IDEPROBE_TEST_CASE")
-    path = Paths.get(screenshotsDir).resolve(testSuite).resolve(testCase)
-  } yield Files.createDirectories(path)
+  private val testSuiteName = sys.env.getOrElse("IDEPROBE_TEST_SUITE", "")
+  private val testCaseName = sys.env.getOrElse("IDEPROBE_TEST_CASE", "")
+  private val screenshotsDir = sys.env.getOrElse("IDEPROBE_SCREENSHOTS_DIR", "/tmp/ide-probe/screenshots")
```

Comment:
I think this env var will always be set by InstalledIntelliJ based on paths config. Resolving default path for screenshots is handled there.

Reply:
Oh! makes sense, Ola pls verify if that's indeed the case